templates/compose/sparkyfitness.yaml

```
@@ -0,0 +1,57 @@
+# documentation: https://codewithcj.github.io/SparkyFitness/
+# slogan: SparkyFitness is a comprehensive fitness tracking and management application designed to help users monitor their nutrition, exercise, and body measurements. It provides tools for daily progress tracking, goal setting, and insightful reports to support a healthy lifestyle.
+# category: health
+# tags: sparkyfitness, fitness, health, nutrition, exercise, body measurements
+# logo: svgs/sparkyfitness.svg
+# port: 80
+
+services:
+  sparkyfitness-frontend:
+    image: 'codewithcj/sparkyfitness:latest'
```

Please pin to a static version rather than 'latest' to prevent the template from breaking due to upstream changes.
```suggestion
    image: 'codewithcj/sparkyfitness:v0.15.7.3' # Released on Oct 18, 2025
```